pkg/rollup/config.go

```diff
@@ -0,0 +1,82 @@
+package rollup
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/jackc/pgx/v4"
+
+	"github.com/timescale/promscale/pkg/internal/day"
+	"github.com/timescale/promscale/pkg/log"
+)
+
+type Config struct {
+	Enable               *bool `yaml:"enabled,omitempty"`
+	UseDefaultResolution *bool `yaml:"use_default_resolution"`
+	Resolution           `yaml:"resolutions,omitempty"`
```

Comment:
This contains multiple resolutions `Info`, I think the plural form suits it better. Also, it differentiates it from `Info.Resolution`.

```suggestion
	Resolutions           `yaml:"resolutions,omitempty"`
```